sql/mozfun/norm/vpn_attribution/udf.sql

```diff
@@ -31,19 +31,22 @@ RETURNS STRUCT<
           -- strip prefix from google and yahoo patterns
           "(?:www[.]|search[.])?(.*)"
         ) AS organic_referrer,
+        STRUCT(
+          COALESCE(utm_campaign, "(not set)") AS campaign,
+          COALESCE(utm_content, "(not set)") AS content,
+          COALESCE(utm_medium, "(none)") AS medium,
+          COALESCE(utm_source, "(direct)") AS source
+        ) AS utm,
```

Comment:
this is used to ensure that the normalized attribution values are never `NULL`, so that `funnel_ga_to_subscriptions_v1/query.sql` can correctly join using normalized attribution. The replacement values match what is provided by GA exports.